modules/create-a-custom-permission-set.adoc

```diff
@@ -9,7 +9,7 @@
 You can create new permission sets from the *Access Control* view.
 
 .Prerequisites
-* You must have the *Admin* role, or a role with the permission set with read and write permissions for the `AuthProvider` and `Role` resources to create, modify, and delete permission sets.
+* You must have the *Admin* role, or a role with the permission set with read and write permissions for the `Access` resource to create, modify, and delete permission sets.
```

Comment:
While we are making changes, could we use the same wording that we use in other places to make this less wordy/awkward?

Example: You must have the *Admin* role, or read and write permissions for the `Access` resource to create, modify, and delete custom roles.

(Just making sure it would be accurate to rewrite it this way - it's definitely easier to understand, IMO.)